app/views/idv/session_errors/failure.html.erb

```diff
@@ -16,6 +16,11 @@
           text: t('idv.troubleshooting.options.contact_support', app_name: APP_NAME),
           new_tab: true,
         },
+        IdentityConfig.store.in_person_proofing_enabled && {
+          url: idv_in_person_url,
+          text: t('in_person_proofing.headings.welcome'),
+          new_tab: true,
```

Comment:
Seems like this should avoid opening a new tab, since it's staying in the application?

Same applies for changes in other views.

```suggestion
```